### PR TITLE
Add UserGuide, Portfolios for Jeff, Jia Xian

### PR DESCRIPTION
**Pull request by koh-jx**
Add the "Add task" function to the UserGuide, also fixed some issues with the images in AboutUs.java not displaying as intended.

Completed Portfolios for Jeff Sieu and Koh Jia Xian

### CODE REVIEW

File: docs/team/jeffsieu.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Technologies: ReactJS, Flutter
+
+## Work Experience
+
+NA (Enlisted in the RSAF)   
+
+## Projects and Competitions
+
+#### [Multivus](https://github.com/wlren/multiverse)
```
Comment (jeffsieu) [inline docs/team/jeffsieu.md:19]: multiverse*

**Review by jeffsieu — CHANGES_REQUESTED** [review]

File: docs/team/jeffsieu.md (hunk 1 above)
Comment (koh-jx) [reply docs/team/jeffsieu.md:19]: bruh

**Review by jeffsieu — APPROVED** [review]
Is Jonathan's supposed to be empty?

**Comment by koh-jx** [comment]
> Is Jonathan's supposed to be empty?

Yes, his portfolio hasn't been provided yet.

**Comment by mingyi456** [comment]
So I guess we wait a bit more for @yeppog to provide his portfolio before merging?

**Comment by wlren** [comment]
Merging this, will do up my portfolio and finish up on UserGuide later thi week

**Pull request merged**